<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1211</width>
    <height>1129</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MorseMesh</string>
  </property>
  <property name="styleSheet">
   <string notr="true">qdarkstyle</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>700</width>
     <height>600</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <layout class="QGridLayout" name="whole_grid">
      <item row="0" column="0">
       <widget class="QFrame" name="frame_vtk">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>400</height>
         </size>
        </property>
        <property name="acceptDrops">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QVTKRenderWindowInteractor" name="vtkWidget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>400</width>
             <height>400</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QFrame" name="frame_sidebar">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>400</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="acceptDrops">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="sidebar_layout">
         <item row="0" column="0">
          <widget class="QGroupBox" name="sidebar">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>400</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="title">
            <string>Parameter Settings</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QFrame" name="frame_bottom">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>650</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>70</height>
         </size>
        </property>
        <property name="acceptDrops">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="bottom_layout">
         <item row="0" column="0">
          <widget class="QSlider" name="high_thresh_slider">
           <property name="minimumSize">
            <size>
             <width>400</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="high_thresh_text">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>high thresh: </string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QSlider" name="low_thresh_slider">
           <property name="minimumSize">
            <size>
             <width>400</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="low_thresh_text">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>low thresh: </string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1211</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_load_ply"/>
    <addaction name="action_load_feature_vector_file"/>
    <addaction name="action_save_segmentation_label_txt"/>
    <addaction name="action_load_label_txt"/>
   </widget>
   <widget class="QMenu" name="menu_compute">
    <property name="title">
     <string>Compute</string>
    </property>
    <addaction name="action_compute_morse_complex"/>
   </widget>
   <widget class="QMenu" name="menu_segmentations">
    <property name="title">
     <string>Segmentations</string>
    </property>
    <addaction name="action_morse_cells_persistence"/>
    <addaction name="action_cluster"/>
    <addaction name="action_segmentation_method"/>
    <addaction name="action_morse_segementation_ridge_first"/>
    <addaction name="action_cluster_segmentation_method"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_quick_guide"/>
    <addaction name="action_info_contact"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_compute"/>
   <addaction name="menu_segmentations"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_load_ply">
   <property name="text">
    <string>Load ply</string>
   </property>
  </action>
  <action name="action_load_feature_vector_file">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Load feature vector file</string>
   </property>
  </action>
  <action name="action_save_segmentation_label_txt">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save segmentation (label txt)</string>
   </property>
  </action>
  <action name="action_load_label_txt">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Load label txt</string>
   </property>
  </action>
  <action name="action_compute_morse_complex">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Compute Morse complex</string>
   </property>
  </action>
  <action name="action_morse_cells_persistence">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Morse cells (persistence)</string>
   </property>
  </action>
  <action name="action_cluster">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Cluster</string>
   </property>
  </action>
  <action name="action_segmentation_method">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Morse segmentation method</string>
   </property>
  </action>
  <action name="action_cluster_segmentation_method">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Cluster segmentation method</string>
   </property>
  </action>
  <action name="action_morse_segementation_ridge_first">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Morse segementation (ridge first)</string>
   </property>
  </action>
  <action name="action_quick_guide">
   <property name="text">
    <string>Quick Guide</string>
   </property>
  </action>
  <action name="action_info_contact">
   <property name="text">
    <string>Info/Contact</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKRenderWindowInteractor</class>
   <extends>QWidget</extends>
   <header>vtk</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
